### ISSUE TITLE
Misc issues in Nistats reports

### ISSUE THREAD

**Issue by bthirion** [nistats/nistats#409]
Here are some  issues I have encountered using nistats reports.
* Does not seem to take into account use-provided parameters for thresholding 
* the contrast maps only have the positive part displayed
* multi-dimensional contrasts do not display well.

**Comment by kchawla-pi** [comment]
> the contrast maps only have the positive part displayed

That's due to issue #362 

> Does not seem to take into account use-provided parameters for thresholding
> multi-dimensional contrasts do not display well.

Examples/code to reproduce?

**Comment by bthirion** [comment]
Some of my comments are wrong. Let me correct them:
- there is a slight inconsistency in the default parameters. default alpha should 0.001 not 0.01, to match the default threshold (3.09)
- #362 
- the display of multi-dimensional contrasts is not good.

Clearly these 3 things are distinct and thus should be handled in separate PRs. I Can work on it.

**Comment by danjgale** [comment]
I've also encountered an issue with the reports. If the inputs to your second-level analysis are a list of  `FirstLevelModel` instances, there is no way to define first-level contrasts of these models. In this case, defining first-level contrasts are necessary, otherwise `SecondLevelModel.compute_contrasts` will throw an error: 

```
ValueError: If second_level_input was a list of FirstLevelModel, then first_level_contrast is mandatory. It corresponds to the second_level_contrast argument of the compute_contrast method of FirstLevelModel
```. 

It seems to me that if you pass in a second-level model, it only accepts second-level contrasts. Not sure whether or not to make a separate issue with the impending nilearn merge.

**Comment by bthirion** [comment]
Thx for reporting.
I think that this is a sepaarte issue, that should be addressed any how.

**Comment by kchawla-pi** [comment]
> Not sure whether or not to make a separate issue with the impending nilearn merge.

I've made a separate issue. It can be worked on in `nilearn.stats` after the merge.

**Comment by danjgale** [comment]
Thanks @kchawla-pi, I'd be happy to work on it once the merge is done!
